thirdparty/xmlstream/xmlstream_p.h

```diff
@@ -1140,10 +1140,11 @@ bool XmlStreamReaderPrivate::parse()
             case '\n':
                 ++lineNumber;
                 lastLineStart = characterOffset + readBufferPos;
+                [[gnu::fallthrough]];
```

Comment:
Yes, such a comment might be the way to go. As mentioned earlier, this is/was the way lint works.
If and when other compilers give a warning there, we can re-think the approach, but for now this seems te best option.